packages/core/src/App/Containers/NotificationsDialog/notifications-dialog.jsx

```diff
@@ -2,152 +2,154 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import React from 'react';
 import { CSSTransition } from 'react-transition-group';
-import { Button, DesktopWrapper, Icon, MobileDialog, MobileWrapper, ThemedScrollbars, Text } from '@deriv/components';
+import {
+    Button,
+    DesktopWrapper,
+    Icon,
+    MobileDialog,
+    MobileWrapper,
+    ThemedScrollbars,
+    Text,
+    useOnClickOutside,
+} from '@deriv/components';
 import { BinaryLink } from 'App/Components/Routes';
 import { connect } from 'Stores/connect';
-import { localize } from '@deriv/translations';
+import { localize, Localize } from '@deriv/translations';
```

Comment:
Just a question: can we stick to one of these two `localize` function since they perform the same actions?

Reply:
I think we should keep both of them, since we're still using `localize` for `MobileDialog` title, if `Localize` component is used there instead, it will return an `object` instead of `string`.  
`MobileDialog` currently only accepts `string` as the `title` prop type.

Reply:
@akmal-binary thanks for your comment.
it's not a rule but it's cleaner if we use the component instead of the localize function in jsx but to pass it as component props, as @zaki-binary  mentioned we need to use function therefore we need both of them here